Python/optimizer_analysis.c

```diff
@@ -463,7 +463,7 @@ remove_unneeded_uops(_PyUOpInstruction *buffer, int buffer_size)
                     last->opcode == _COPY
                 ) {
                     last->opcode = _NOP;
-                    buffer[pc].opcode = NOP;
+                    buffer[pc].opcode = _NOP;
```

Comment:
`NOP` vs. `_NOP`... I hope the difference between those two is documented somewhere[^1] 😄

[^1]: After a quick `git grep` I could not figure out the semantic difference between them.

Reply:
In Tier 1 it's NOP. In Tier 2 it's `_NOP`. The original code worked because they are *currently* the same value (as are all opcodes that are shared between T1 and T2). But eventually we may want all the Tier 2 opcodes to be distinct.

Reply:
Thanks for the clarification!